src/views/send/index.js

```diff
@@ -168,7 +168,7 @@ class Send extends Component {
             // Disable loader indicator
             this.enableLoaderIndicatory(false);
             // Push to transaction detail page
-            this.pushToTxDetail();
+            this.pushToTxDetail(txResponse.txhash);
```

Comment:
What happens when you get pushed to this page? The tx info probably won't be able to be fetched as the block has not passed, so, wonder what's the expected behavior?